ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2199,6 +2199,21 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-longrun
     workflow: cucushift-installer-rehearse-gcp-ipi-private
+- as: gcp-ipi-proxy-etcd-encryption-f28-compliance
```

Comment:
Is step `etcd-encryption` required for compliance testing ? I can see the other added profiles have this step, but not this profile.

Reply:
Not needed for this one. The workflow cucushift-installer-rehearse-gcp-ipi-proxy-etcd-encryption already enabled it by default.

Reply:
But yes for all the others. A single etcd encryption will cost about 25 minutes. Enable it & disable it in a single test case is too time consuming. We have 6 test cases to get the cluster compliant per different security standards which requires etcd encryption. Thus I enabled it by default.